Assets/Runtime/ScreenTransitions/SimpleFadeTransition.cs

```diff
@@ -20,27 +20,33 @@ public class SimpleFadeTransition : ATransitionComponent
         private Action currentAction;
         private Transform currentTarget;
 
-        private float startValue;
-        private float endValue;
+        private float startValue = 0f;
+        private float endValue = 0f;
 
         private bool shouldAnimate;
 
-        public override void Animate(Transform target, Action callWhenFinished) {
-            if (currentAction != null) {
-                canvasGroup.alpha = endValue;
-                currentAction();
-            }
-
+        public override void Animate(Transform target, Action callWhenFinished)
+        {
+            currentTarget = target;
```

Comment:
`currentTarget` is not used, let remove it

Reply:
done